hack/update-swagger-spec.sh

```diff
@@ -51,16 +51,20 @@ TEMP_DIR=${USE_TEMP:-$(mktemp -d /tmp/openshift-cmd.XXXX)}
 export CURL_CA_BUNDLE="${TEMP_DIR}/openshift.local.config/master/ca.crt"
 
 # Start openshift
+echo "Starting OpenShift..."
 pushd "${TEMP_DIR}" > /dev/null
-OPENSHIFT_ON_PANIC=crash openshift start master --listen="https://0.0.0.0:8443" --master="https://127.0.0.1:8443" 1>&2 &
+OPENSHIFT_ON_PANIC=crash openshift start master --listen="https://0.0.0.0:8443" --master="https://127.0.0.1:8443" >/dev/null 2>&1 &
 OS_PID=$!
 popd > /dev/null
 
 wait_for_url "${HOST}/healthz" "apiserver: " 0.25 80
 
-echo "Updating ${SWAGGER_ROOT_DIR}"
-set -x
+echo "Updating ${SWAGGER_ROOT_DIR}:"
+echo
+echo "Updating ${SWAGGER_ROOT_DIR}/oapi-v1.json from ${SWAGGER_API_PATH}oapi/v1..."
```

Comment:
`set -x` also outputted a bunch of other extraneous commands that was irritating to look at and completely useless.